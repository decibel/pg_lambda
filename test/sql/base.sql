\set ECHO none

\i test/pgxntool/setup.sql

SELECT plan(
  1
);

SELECT is(
  lambda(
    $l$( i1 int, i2 int ) RETURNS int LANGUAGE sql AS 'SELECT i1 + i2'$l$
    , 1::int
    , 2
  )
  , 3
  , 'Test simple addition function'
);


\i test/pgxntool/finish.sql

-- vi: expandtab ts=2 sw=2
