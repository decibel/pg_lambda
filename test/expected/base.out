\set ECHO none
1..1
ok 1 - Test simple addition function
# TRANSACTION INTENTIONALLY LEFT OPEN!
